client/state/audio/test/middleware.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy, stub } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import middleware from '../middleware';
+import { HAPPYCHAT_RECEIVE_EVENT } from 'state/action-types';
+
+describe( 'Audio Middleware', () => {
+	let next;
+	let store;
+	let play;
+
+	beforeEach( () => {
+		next = spy();
+
+		store = {
+			dispatch: spy(),
+			getState: stub(),
+			replaceReducers: spy(),
+			subscribe: spy(),
+		};
+
+		store.getState.returns( Object.create( null ) );
+
+		// Spy on (new Audio()).play()
+		play = spy();
+		global.window = {
+			Audio: spy( function() {
+				return { play };
+			} ),
+		};
+	} );
+
+	afterEach( () => {
+		global.window = undefined;
```

Comment:
I trust you if you say yes, but are we sure this is the right way to reset `global` after the tests?